core-blocks/image/index.js

```diff
@@ -113,8 +116,11 @@ export const settings = {
 					const align = alignMatches ? alignMatches[ 1 ] : undefined;
 					const idMatches = /(?:^|\s)wp-image-(\d+)(?:$|\s)/.exec( className );
 					const id = idMatches ? idMatches[ 1 ] : undefined;
+					const anchorEl = node.querySelector( 'a' );
```

Comment:
Can you rename this `anchorElement` instead of the abbreviation? Two-letter abbreviations are somewhat cryptic and unnecessry 😄